docs/v3/source/includes/resources/routes/_destination_object.md.erb

```diff
@@ -18,10 +18,11 @@ Note that when using a custom port, the app process must be listening on the spe
 
-| Name                        | Type                     | Description                                                                                                                               |
-| --------------              | ------------------------ | ------------------------------------------------------                                                                                    |
-| **guid**                    | _uuid_                   | Unique identifier for the destination                                                                                                    |
-| **app.guid**                | _uuid_                   | Unique identifier for the app to route traffic to                                                                               |
-| **app.process.type**        | _string_                 | Type of the process belonging to the app to route traffic to                                                                      |
-| **port**                    | _integer_                | Port on the destination process to route traffic to
-| **weight** (*experimental*) | _integer_ or `null`      | Percentage of traffic which will be routed to this destination; _this feature is only available via the BETA Service Mesh routing plane_ |
+| Name                          | Type                     | Description                                                                                                                               |
+| --------------                | ------------------------ | ------------------------------------------------------                                                                                    |
+| **guid**                      | _uuid_                   | Unique identifier for the destination                                                                                                    |
+| **app.guid**                  | _uuid_                   | Unique identifier for the app to route traffic to                                                                               |
+| **app.process.type**          | _string_                 | Type of the process belonging to the app to route traffic to                                                                      |
+| **port**                      | _integer_                | Port on the destination process to route traffic to
+| **weight** (*experimental*)   | _integer_ or `null`      | Percentage of traffic which will be routed to this destination; _this feature is only available via the BETA Service Mesh routing plane_ |
+| **protocol** (*experimental*) | _string_                 | Protocol to use, valid protocols are `http1`, `http2`, or `tcp`; _this feature is only available if you turn it on in routing release_ |
```

Comment:
based on the tests below, `null` is a valid value, too--or am I missing something?  Should that behavior be clarified here?